src/inputs/getDynamicOptions.ts

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+import { Disposable, window } from "vscode";
+
+import { CancelError, NoNameGivenError } from "../errors";
+import { IMultiStepData } from "./getUserInput";
+
+export async function getDynamicOptions(multiStep: IMultiStepData): Promise<string> {
```

Comment:
I think we should have this function return an array of strings -- let's do the splitting in here so that we can change this implementation in the future.

Then throughout the app we would be passing an array of dynamic option strings instead of the semi-colon delimited string. Would make it easier in the future to change how the users enters multiple dynamic options.

Reply:
Good call, I'll make it so.